### DISCUSSION TITLE
[Avatar] Create walking animation from image

### DISCUSSION THREAD

**Discussion by UnDVD2Euros** [pixijs/pixijs#8139] [Q&A]
I currently have several images to build a 2D character Hat, Body, Face and Shoes. I manage to place each part in the order I want correctly. Now I am looking for a solution since the shoes have no animation, they are only a simple image. Is there a possibility with PIXI to create an animation from a simple image to create an animation like walking.

![seq_0001](https://user-images.githubusercontent.com/97616038/151055485-1576e0c2-42fc-4d55-8b63-fbcc56f34602.png)
There is my simple image for the left and right shoe of the character.

Can we create several empty frames using this image to rotate and change Y position to make it like walking animation.
Also i woudd like to use the animation as a wrapper so i can replace the shoe texture by another one.

**Comment by andrewstart** [comment]
You would be looking for an animation tool (unless you want to set up a whole bunch of tweens through code). The most easily implemented ones would be [Spine](http://esotericsoftware.com/) using [pixi-spine](https://github.com/pixijs/spine) or Adobe Animate ([and our extension](https://github.com/pixijs/animate-extension)) using [pixi-animate](https://github.com/pixijs/animate). Spine has built in support for swapping out skins on your skeleton, and replacing textures in Animate is doable, but would require a little bit of planning.
